### ISSUE TITLE
Using testjar xmlpathinjar fails to perform suite, single tests do work

### ISSUE THREAD

**Issue by escay** [cbeust/testng#1855]
### TestNG Version
Maven: org.testng:testng:6.11

### Expected behavior
When testng.xml refers to a suite.xml and the xml files are packaged in a jar the -testjar and -xmlpathinjar should work. But it only works when I am not using suites.

### Actual behavior
java -Xms512m -Xmx512m -XX:MaxMetaspaceSize=256m -cp .;./target/lib/* org.testng.TestNG -testjar ./target/lib/testclient.common-tests.jar -xmlpathinjar testng.xml
[TestNG] [ERROR] java.io.IOException: Stream closed

when I use:

```
<?xml version="1.0"?>
<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >
<suite name="Hello Tests" parallel="false">
    <suite-files>
        <suite-file path="./hello-tests.xml" />
    </suite-files>
</suite>
```
it fails with [TestNG] [ERROR] java.io.IOException: Stream closed.
when I use:

```
<?xml version="1.0"?>
<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >

    <test name="Hello test">
                <classes>
            <class name="com.hello.SomeTest" />
        </classes>
    </test>
```
the test executes correctly and does not fail with [TestNG] [ERROR] java.io.IOException: Stream closed.
The jar file contains hello-tests.xml next to the testng.xml file in the root of the jar file.

It seems like the reference to the suite file is not found.
I also tested:

`        <suite-file path="hello-tests.xml" />`
instead of 
`        <suite-file path="./hello-tests.xml" />`
but this did not help.

### Test case sample
See above.

### Reference
Seems like someone else had the same problem in the past:
http://testng.1065351.n5.nabble.com/Run-suite-of-suites-using-command-line-td21765.html

**Comment by krmahadevan** [comment]
Can you please retry using the latest released version of TestNG (6.14.3 as
of today) and let us know if this is still a problem ?

>

**Comment by krmahadevan** [comment]
Closing this issue based on @ken-p feedback.
